diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,6 +16,7 @@ endif()

 # Set the project and version
 project(c3c VERSION ${CMAKE_MATCH_1} LANGUAGES C CXX)
+set(CMAKE_SKIP_RPATH ON)
 message("Configuring C3C ${CMAKE_PROJECT_VERSION} for ${CMAKE_SYSTEM_NAME}")

 # Helper functions
@@ -280,7 +281,7 @@
     list(GET VERSION_LIST 0 LLVM_MAJOR_VERSION)

     include_directories(${LLVM_INCLUDE_DIRS})
-    link_directories(${LLVM_LIBRARY_DIRS})
+    # link_directories(${LLVM_LIBRARY_DIRS})
     add_definitions(${LLVM_DEFINITIONS})

     if(NOT C3_LINK_DYNAMIC)
